Anature/src/application/controllers/ClickQueue.java

```diff
@@ -5,7 +5,7 @@
 	private class Node
 	{
 		private Runnable mData;
-		private Node mNext;
+		public Node mNext;
```

Comment:
This doesn't have to be public since it is an inner class that will only ever be used by the ClickQueue.